Libraries/LibWeb/CSS/StyleValues/CSSColorValue.cpp

```diff
@@ -17,14 +17,15 @@
 
 namespace Web::CSS {
 
-ValueComparingNonnullRefPtr<CSSColorValue> CSSColorValue::create_from_color(Color color)
+ValueComparingNonnullRefPtr<CSSColorValue> CSSColorValue::create_from_color(Color color, Optional<FlyString> name)
 {
-    auto make_rgb_color = [](Color const& color) {
+    auto make_rgb_color = [name](Color const& color) {
         return CSSRGB::create(
             NumberStyleValue::create(color.red()),
             NumberStyleValue::create(color.green()),
             NumberStyleValue::create(color.blue()),
-            NumberStyleValue::create(color.alpha() / 255.0));
+            NumberStyleValue::create(color.alpha() / 255.0),
+            name);
```

Comment:
According to [the spec](https://drafts.csswg.org/css-color-4/#serializing-sRGB-values): 
> When serializing the value of a property which was set by the author to a CSS [named color](https://drafts.csswg.org/css-color-4/#named-color), a [system color](https://drafts.csswg.org/css-color-4/#css-system-colors), a [deprecated-color](https://drafts.csswg.org/css-color-4/#deprecated-system-colors), or [transparent](https://drafts.csswg.org/css-color-4/#valdef-color-transparent) therefore, for the [declared value](https://drafts.csswg.org/css-cascade-5/#declared-value), the [ASCII lowercase](https://infra.spec.whatwg.org/#ascii-lowercase) keyword value is retained.

So this needs to be converted to ASCII lowercase at some point. It may be best to do that in `CSSRGB::to_string()` below.